src/Host.cpp

```diff
@@ -178,10 +178,11 @@ Host::Host(int port, const QString& hostname, const QString& login, const QStrin
     // file auditing and other issues)
     mErrorLogStream.setDevice(&mErrorLogFile);
 
-    // There was a map load attempt made here but it did not seem to be needed
-    // and it caused issues being doing in the constructor (some other classes
-    // were not fully initialised at this point) so it seemed sensible to remove
-    // it - Slysven
```

Comment:
This seems simple enough, but I'm concerned about the issues this functionality previously caused, as remarked in the code comment that was made here by @SlySven   

When I tested this initially with Lusternia, I was prompted to either download the map or make my own.  I selected to download the map and this seemed to work alright up until the "Assigning rooms to their areas..." phase and then I get a seg fault crash.  I don't get the same results with this commented out and opening the map manually.   
I should note that I don't have any previous data/settings for the "lusternia" profile though I'm not sure if that is a major factor in the crash conditions.